libs/core/src/components.d.ts

```diff
@@ -233,39 +233,39 @@ export namespace Components {
          */
         "componentId": string;
         /**
-          * Indicates whether or not the input field is disabled
+          * If true, the input field is disabled.
```

Comment:
Could we say something more explanatory, like this
```suggestion
          * If true, the user cannot interact with the field.
```